### ISSUE TITLE
[Gender Bias] Genderation bias task parsing error 

### ISSUE THREAD

**Issue by gabrielle-lau** [facebookresearch/ParlAI#3659]
**Bug description**
I'm getting `AttributeError: 'NoneType' object has no attribute 'split'` when running the genderation task command  `parlai display_data -t genderation_bias:controllable_task:convai2`. The "task" key's value is None in the opt dict. 

**Reproduction steps**
`parlai display_data -t genderation_bias:controllable_task:convai2`

GitHub page for task: [https://github.com/facebookresearch/ParlAI/tree/47765c3256a3e91ba4ae50fdc42250ad5f0c0ecf/parlai/tasks/genderation_bias](https://github.com/facebookresearch/ParlAI/tree/47765c3256a3e91ba4ae50fdc42250ad5f0c0ecf/parlai/tasks/genderation_bias)

**Expected behavior**
It should display the convai2 data labelled with gender control variables eg. "f1m0" :

```- - - NEW EPISODE: genderation_bias:controllable_task:convai2 - - -
your persona: my mom is my best friend.
your persona: i have four sisters.
your persona: i believe that mermaids are real.
your persona: i love iced tea.
hi , how are you doing today ? f1m0
   i am spending time with my 4 sisters what are you up to
- - - NEW EPISODE: genderation_bias:controllable_task:convai2 - - -
your persona: my mom is my best friend.
your persona: i have four sisters.
your persona: i believe that mermaids are real.
your persona: i love iced tea.
hi , how are you doing today ?
i am spending time with my 4 sisters what are you up to
wow , four sisters . just watching game of thrones . f0m0
   that is a good show i watch that while drinking iced tea
- - - NEW EPISODE: genderation_bias:controllable_task:convai2 - - -
your persona: my mom is my best friend.
your persona: i have four sisters.
your persona: i believe that mermaids are real.
your persona: i love iced tea.
hi , how are you doing today ?
i am spending time with my 4 sisters what are you up to
wow , four sisters . just watching game of thrones .
that is a good show i watch that while drinking iced tea
i agree . what do you do for a living ? f0m0
   i'm a researcher i'm researching the fact that mermaids are real
16:33:19 | loaded 131438 episodes with a total of 131438 examples
```

**Logs**
Command line output:

```
Traceback (most recent call last):
  File "/home/user/project/bin/parlai", line 10, in <module>
    sys.exit(main())
  File "/home/user/project/lib/python3.8/site-packages/parlai/__main__.py", line 14, in main
    superscript_main()
  File "/home/user/project/lib/python3.8/site-packages/parlai/core/script.py", line 297, in superscript_main
    opt = parser.parse_args(args)
  File "/home/user/project/lib/python3.8/site-packages/parlai/core/params.py", line 1153, in parse_args
    self.add_extra_args(args)
  File "/home/user/project/lib/python3.8/site-packages/parlai/core/script.py", line 156, in add_extra_args
    sa.choices[args[0]].add_extra_args(args)
  File "/home/user/project/lib/python3.8/site-packages/parlai/core/params.py", line 930, in add_extra_args
    self.add_task_args(task, partial)
  File "/home/user/project/lib/python3.8/site-packages/parlai/core/params.py", line 842, in add_task_args
    agent.add_cmdline_args(self, partial)
  File "/home/user/project/lib/python3.8/site-packages/parlai/tasks/genderation_bias/agents.py", line 96, in add_cmdline_args
    tasks = get_original_task_module(opt, multi_possible=True)
  File "/home/user/project/lib/python3.8/site-packages/parlai/tasks/genderation_bias/utils.py", line 148, in get_original_task_module
    tasks = opt['task'].split(',')
AttributeError: 'NoneType' object has no attribute 'split'
```

**Versions**
parlai 1.2.0
Python 3.8.2

Appreciate any help! Thank you.

**Comment by stephenroller** [comment]
@emilydinan

**Comment by emilydinan** [comment]
Hi @gabrielle-lau I cannot repro the error on master. Can you pull and try again?

**Comment by gabrielle-lau** [comment]
Thanks for your help. I pulled again but I got the same error using `parlai display_data -t genderation_bias:controllable_task:convai2`. Surprisingly, what made the parsing error go away is to change `-t` to `--task`. For future reference, my working command is:
```parlai display_data --task genderation_bias:controllable_task:convai2```
which gives the expected output (ConvAI2/train_self_original.txt with genderation annotations).
